### PR TITLE
New boards and test updates

### PR DESCRIPTION
**Pull request by c1728p9**

### CODE REVIEW

**Comment by c1728p9** [comment]
test this please

**Comment by c1728p9** [comment]
test this please

File: pyOCD/flash/flash_lpc824.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+"""
+ Flash OS Routines (Automagically Generated)
```
Comment (0xc0170) [inline pyOCD/flash/flash_lpc824.py]: Automagically  :smile:

**Comment by 0xc0170** [comment]
LGTM

Once test passing, we shall merge and release a new minor version .

**Comment by c1728p9** [comment]
test this please

**Comment by c1728p9** [comment]
test this please

**Comment by c1728p9** [comment]
test this please

**Comment by c1728p9** [comment]
test this please

**Comment by c1728p9** [comment]
test this please

**Comment by bridadan** [comment]
test this please

**Comment by bridadan** [comment]
test this please

**Comment by c1728p9** [comment]
test this please

**Comment by bridadan** [comment]
test this please

**Comment by 0xc0170** [comment]
+1 for having the summarY!

**Pull request merged**